ext/pdo/pdo.c

```diff
@@ -281,6 +281,40 @@ PDO_API zend_string *php_pdo_int64_to_str(int64_t i64) /* {{{ */
 }
 /* }}} */
 
+/* Convert uint64 to zend_string */
+PDO_API zend_string *php_pdo_uint64_to_str(uint64_t i64) /* {{{ */
+{
+	char buffer[65];
+	char outbuf[65] = "";
+	register char *p;
+	zend_long long_val;
+	char *dst = outbuf;
+
+	if (i64 == 0) {
+		return ZSTR_CHAR('0');
+	}
+
+	p = &buffer[sizeof(buffer)-1];
+	*p = '\0';
+
+	while ((uint64_t)i64 > (uint64_t)ZEND_LONG_MAX) {
```

Comment:
```suggestion
	while (i64 > (uint64_t)ZEND_LONG_MAX) {
```
i64 is already an `uint64_t`